test/reducer.spec.js

```diff
@@ -32,7 +32,7 @@ describe('#reducer', () => {
         const expectedState = defaultState.merge({
           [defaultState.activeTabIndex]: {
             activeRoute: { name: 'ROUTE_A', params: { routeParam: 4 } },
-            method: 'push',
+            method: actions.PUSH,
```

Comment:
doesn't this break everything when you do `navigator[navigationMethod]()` in here https://github.com/Wolox/react-native-renavigate/pull/6/files#diff-0fdff0a4ab22c86a59957a77d774ecceR52